actions/merger/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+# config
+GITHUB_REPO=${GITHUB_REPO:-jade-data-repo}
+GITHUB_USER_EMAIL=${GITHUB_USER_EMAIL:-broadbot@broadinstitute.org}
+GITHUB_USER_NAME=${GITHUB_USER_NAME:-broadbot}
+GITHUB_WORKSPACE=${GITHUB_WORKSPACE}
+COMMIT_MESSAGE="${COMMIT_MESSAGE:-"commit merged via github action"}"
+MERGE_BRANCH=${MERGE_BRANCH:-develop}
+SWITCH_DIRECTORIES=${SWITCH_DIRECTORIES:-"false"}
```

Comment:
Not that it makes a difference, but some items have `"` here and some do not. They're equivalent but the difference may make a reader think that there's a reason for them to be different.